src/ccl_correlation.c

```diff
@@ -213,19 +184,14 @@ static void ccl_tracer_corr_bessel(ccl_cosmology *cosmo,
   gsl_integration_workspace *w = NULL;
   int local_status;
 
-  #pragma omp parallel default(none) \
-                       shared(cosmo, status, wtheta, n_ell, ell, cls, \
-                              corr_type, cl_spl, theta, n_theta) \
-                       private(w, F, result, eresult, local_status, ith, \
-                               gslstatus, cp)
+#pragma omp parallel default(none)				      \
+  shared(cosmo, status, wtheta, n_ell, ell, cls,		      \
+	 corr_type, cl_spl, theta, n_theta)				\
+  private(w, F, result, eresult, local_status, ith,			\
+	  gslstatus, cp)
```

Comment:
Please preserve the visual spacing here.

Reply:
This was the automatic spacing emacs gave me. I've gone back to the previous one.

Reply:
please turn off tabs! ;p